src/main/java/uk/gov/companieshouse/psc/extensions/api/mongo/document/Data.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.psc.extensions.api.mongo.document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Data {
```

Comment:
Class should also override equals and hashCode as it makes up part of the PscExtension entity.